packages/volto/src/components/theme/Logo/__snapshots__/Logo.Multilingual.test.jsx.snap

```diff
@@ -2,52 +2,50 @@
 
 exports[`Multilingual Logo renders a logo component in a multilingual site language root 1`] = `
 <img
-  alt="English"
+  alt="Logo of Plone Site"
   className="ui image"
   src="Logo.svg"
-  title="English"
 />
 `;
 
 exports[`Multilingual Logo renders a logo component in a multilingual site root 1`] = `
 <a
+  aria-label="Home"
   className={null}
   href="/en"
   onClick={[Function]}
   target={null}
-  title="Plone Site"
+  title={null}
 >
   <img
-    alt="Plone Site"
+    alt="Logo of undefined"
     className="ui image"
     src="Logo.svg"
-    title="Plone Site"
   />
 </a>
 `;
 
 exports[`Multilingual Logo renders a logo component with a custom logo in a non-root url 1`] = `
 <img
-  alt="English"
+  alt="Logo of undefined"
   className="ui image"
   src="/@@site-logo/logo.cab945d8.svg"
-  title="English"
 />
 `;
 
 exports[`Multilingual Logo renders a logo component with a custom logo in a non-root url with path 1`] = `
 <a
+  aria-label="Home"
   className={null}
   href="/en"
   onClick={[Function]}
   target={null}
-  title="English"
+  title={null}
 >
   <img
-    alt="English"
+    alt="Logo of undefined"
```

Comment:
Can we implement the mock so the test does not throw "undefined"?

Reply:
Done